editoast/src/views/v2/train_schedule.rs

```diff
@@ -846,91 +831,4 @@ mod tests {
             update_train_schedule_form.train_schedule.rolling_stock_name
         )
     }
-
-    #[rstest]
-    #[ignore] // TODO: This test should be rewritten using mocks
-    async fn train_schedule_simulation(
```

Comment:
Why is this test removed? Was ignored in https://github.com/OpenRailAssociation/osrd/pull/7165 but feels like we should do something about it (something else than removing it).

Same for the test below.

Reply:
There's an issue open on that : https://github.com/OpenRailAssociation/osrd/issues/8121. I would say that it's fine to delete them now and fetch them in the git history when we'll work on them again. Wdyt?

Reply:
OK, if we track that somewhere else, fine by me!